src/Util.cpp

```diff
@@ -751,7 +751,7 @@ void run_with_large_stack(const std::function<void()> &action) {
     // as opposed to having them cause silent corruption. We pick an
     // amount of memory that should be comfortably larger than most
     // stack frames - 64k.
-    const size_t guard_band = 64 * 1024;
+    const size_t guard_band = (size_t)(64 * 1024);
```

Comment:
I also think you ought to be able to a size_t as the product of two int literals.

Reply:
Yep, this is clearly just a dumb mistake on the part of this version of clang-tidy. One would think they'd have caught this.